src/System.Management.Automation/engine/CoreAdapter.cs

```diff
@@ -4661,6 +4662,221 @@ protected override PSMemberInfoInternalCollection<T> GetMembers<T>(object obj)
     }
 
     /// <summary>
+    /// Deals with DataRow objects
```

Comment:
Oh, I see, you just moved this type, but it's hard to see that for sure. Can it stay in the same place to make it easier to see if anything changed - and if it should move, do that in another refactoring PR?